recipes/mcroni/meta.yaml

```diff
@@ -1,39 +1,46 @@
-{% set version = "1.0.1" %}
-{% set sha256 = "584afa62047ab9598b8fd89d0d39fbd265baf402103005d5efbbb71b440812a5" %} 
+{% set version = "1.0.0" %}
+{% set sha256 = "3e7c008f719a8447446d2706c8738beeb4975b1f" %}
+
 
 package:
   name: mcroni
   version: {{ version }}
 
 source:
-  url: https://github.com/liampshaw/mcroni/archive/v{{ version }}.tar.gz
+  url: https://github.com/liampshaw/mcroni/archive/{{ version }}.tar.gz
+
   sha256: {{ sha256 }}
 
 build:
   number: 0
   noarch: python
   entry_points:
     - mcroni = mcroni.mcroni:main
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
```

Comment:
I think you introduce some whitespace by accident. I check before committing always with "git diff"  to see if I introduced these changes.

Reply:
thanks for this and other comments! coming back to this after a while away to see if I can actually successfully merge or not